src/DynamoCore/Models/DynamoModel.cs

```diff
@@ -3156,6 +3156,12 @@ private void OnWorkspacePropertyChanged(object sender, PropertyChangedEventArgs
             if (args.PropertyName == "EnablePresetOptions")
                 OnPropertyChanged("EnablePresetOptions");
         }
+
+        public PreferenceSettings ImportPreferences(string newSettingsPath)
+        {
+            return PreferenceSettings.Load(newSettingsPath);
+        }
```

Comment:
Wrong section, this is the section of private functions. Also I think this can be internal method

Reply:
Done